yarn.lock

```diff
@@ -10997,9 +10997,9 @@ prism-react-renderer@^1.3.5:
   integrity sha512-IJ+MSwBWKG+SM3b2SUfdrhC+gu01QkV2KmRQgREThBfSQRoufqRfxfHUxpG1WcaFjP+kojcFyO9Qqtpgt3qLCg==
 
 prismjs@^1.28.0:
-  version "1.28.0"
-  resolved "https://registry.yarnpkg.com/prismjs/-/prismjs-1.28.0.tgz#0d8f561fa0f7cf6ebca901747828b149147044b6"
-  integrity sha512-8aaXdYvl1F7iC7Xm1spqSaY/OJBpYW3v+KJ+F17iYxvdc8sfjW194COK5wVhMZX45tGteiBQgdvD/nhxcRwylw==
+  version "1.29.0"
```

Comment:
minor update, the only impact I found in my visual tests was this change in Java code blocks:

![CleanShot 2023-08-25 at 15 05 55@2x](https://github.com/facebook/react-native-website/assets/749374/834df4c2-a136-46f0-bf5a-edeefad84b17)

- Before: https://reactnative.dev/docs/headless-js-android#example-usage
- After: https://deploy-preview-3830--react-native.netlify.app/docs/headless-js-android#example-usage